### PR TITLE
Preprod: movie: Chaos Walking (2021) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
I sense the start of a lengthy film franchise. Rating 3/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 1]
```diff
@@ -799,6 +799,49 @@ sacrifice for one's people [Admiral Adams blew himself up, taking out a number o
...
+:: Title
+Chaos Walking
+
+:: Description
+Chaos Walking is a 2021 American dystopian action film directed by Doug Liman
+from a screenplay by Patrick Ness and Christopher Ford. It is based on Ness's
+science fiction trilogy Chaos Walking, adapting its first book, 2008's The
+Knife of Never Letting Go.
+
+Synopsis: A young man lives in a dystopian world without women, where all
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:814]: As we've noted before: the line between dys- and u-topia is hair fine.

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 2]
```diff
@@ -799,6 +799,49 @@ sacrifice for one's people [Admiral Adams blew himself up, taking out a number o
...
+:: References
+https://en.wikipedia.org/wiki/Chaos_Walking_(film)
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+space colonization [The story is set in and around an all-male colony on the distant planet New World.]
+biased sex-ratio society [The story is explores life on an all-male space colony.]
+what if I heard all the thoughts around me [A main novelty of the film is that the New World males could hear each other's thoughts in streams of images, words, and sounds called "Noise".]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:831]: https://www.themeontology.org/theme.php?name=telepathic%20society springs to mind...

**Review by odinlake — APPROVED** [review]
One suggestion inline

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt:831]: Good call. Added it.

**Pull request merged**